### DISCUSSION TITLE
Need help in pushing stream as WebRTC and playing back as HLS & RTMP push

### DISCUSSION THREAD

**Discussion by rushikimedia** [OvenMediaLabs/OvenMediaEngine#1103] [Help]
Hi,

We are trying to user the WEBRTC stream as the source and trying to play stream in HLS and also trying to Push steams to RTMP target.

we user following outputProfile for that

{
  "encodes": {
      "audios": [
          {
              "bitrate": "128000",
              "channel": 2,
              "codec": "aac",
              "samplerate": 48000
          },
          {
              "bypass": true,
              "name": "bypass_audio"
          }
      ],
      "videos": [
          {
              "bypass": true,
              "name": "bypass_video"
          }
      ]
  },
  "name": "aacCodec",
  "outputStreamName": "${OriginStreamName}_aac"
}

But we are facing issue in playback and RTMP push.

Am I doing anything Wong?

Can anyone suggest the correct Output Profile for this requirement?

**Comment by rushikimedia** [comment]
![image](https://user-images.githubusercontent.com/61576837/208244743-d77c07c1-8756-4f05-af47-2744c908bb3a.png)

After some time of WEBRTC Vidio push, I am getting this Error. Then RTMP push Audio and Video start breaking(on target side)

**Comment by getroot** [comment]
The stability of webrtc ingest has been greatly improved in the latest master version. Test with the latest master version. Also, WebRTC input -> HLS output may require video encoding, not bypass.
